server/chat.ts

```diff
@@ -1293,17 +1293,17 @@ export class CommandContext extends MessageContext {
 					if ((!this.room || this.room.settings.isPersonal || this.room.settings.isPrivate === true) && !this.user.can('lock')) {
 						const buttonName = / name ?= ?"([^"]*)"/i.exec(tagContent)?.[1];
 						const buttonValue = / value ?= ?"([^"]*)"/i.exec(tagContent)?.[1];
-						if (buttonName === 'send' && buttonValue?.startsWith('/msg ')) {
-							const [pmTarget] = buttonValue.slice(5).split(',');
+						if (buttonName === 'send' && (buttonValue?.startsWith('/msg ') || buttonValue?.startsWith('/pm '))) {
+							const [pmTarget] = buttonValue.replace(/^\/(?:msg|pm) /, '').split(',');
```

Comment:
```suggestion
						const msgCommandRegex = /^\/(?:msg|pm|w) /;
						if (buttonName === 'send' && msgCommandRegex.test(buttonValue || '')) {
							const [pmTarget] = buttonValue.replace(msgCommandRegex, '').split(',');
```

Reply:
Right, I didn't think of that. Thanks. n.n